pom.xml

```diff
@@ -170,6 +191,46 @@
         </pluginRepository>
       </pluginRepositories>
     </profile>
+    <profile>
+      <id>override-fkc</id>
+      <activation>
+        <property>
+<!-- We need to activate the profile by using the opposite of the property used for the default profiles -->
+          <name>default</name>
```

Comment:
See the comment just above: this activation property needs to be the opposite of the one used in the default profile so that we can deactivate the default one when we want to use this profile. I'm all in favor of a better solution if you can think of one :smile:

Reply:
And this is how it's used: https://github.com/quarkiverse/quarkus-operator-sdk/pull/945/files#diff-5a69d1f9d9722e35505038f1e9859d2e85c048cc2c85a2f47e1d253bdf57c70cR15